server/routes/getAllTodos.js

```diff
@@ -7,7 +7,11 @@ module.exports = async function getAllTodos (req, res) {
     // const { docs } = await hyper.data.list()
     // const todos = docs.filter(item => item.type === 'todo')
     const { docs } = await hyper.data.query({ type: 'todo' })
-    res.json(docs)
+    if (docs) {
```

Comment:
`docs` will come back as an empty array, so this null check will always pass. You probably want to check for `docs.length` or `docs && docs.length`

Reply:
Ok. I will make that change.